migrating_to_2.0/general.rst

```diff
@@ -54,6 +54,15 @@ Extensions
 
 Work in progress
 
+Hooks
+^^^^^
+
+- Hooks folder has been updated to ``~/.conan2.extensions/hook``;
+- Any hook file must be named with ``hook_`` as prefix and ``.py`` as suffix;
+- Only ``ConanFile`` is passed as parameter;
+- Pre and Post Download are no longer supported in Conan 2.x
```

Comment:
what's the replacement?

Reply:
I do not know the significance, I just stole this from a PR to help :)

Reply:
There are no replacement. Hooks are recipe pre/post methods.
Package signing checks happens post-download too. Other cases need to be discussed, but they will be plugins, not hooks (not necessarily having ``conanfile`` as an argument)

Reply:
Thanks for the details 👍